src/test-helpers/getThoughtFromDB.ts

```diff
@@ -0,0 +1,10 @@
+import * as db from '../data-providers/dexie'
+import dbGetThought from '../data-providers/data-helpers/getThought'
```

Comment:
I believe this file should go in `/src/test-helpers/e2e-helpers` since it is embedded in the puppeteer environment vs being called directly in test code.

Reply:
done

Reply:
👍🏻